book/07-git-tools/sections/rewriting-history.asc

```diff
@@ -292,10 +292,10 @@ $ git log -4 --pretty=format:"%h %s"
 1c002dd Add cat-file
 9b29157 Add blame
 35cfb2b Update README formatting
-f3cc40e Change my name a bit
+f7f3f6d Change my name a bit
 ----
 
-Once again, this changes the SHA-1s of all the commits in your list, so make sure no commit shows up in that list that you've already pushed to a shared repository.
+This changes the SHA-1s of the three most recent commits in your list, so make sure no changed commit shows up in that list that you've already pushed to a shared repository.  Notice that the last commit (`f7f3f6d`) in the list is unchanged.  Despite this commit being shown in the script, because it was marked as ``pick'' and was applied prior to any rebase changes, Git leaves the commit unmodified.
```

Comment:
Great! Let's just format this to abide by the one-sentence-per-source-line convention and we cacn get it merged!